dev-tools/compose/docker-compose.core.yml

```diff
@@ -5,6 +5,8 @@ services:
       context: ${PROJECT_ROOT}/core
       dockerfile: ./infra/Dockerfile
       target: final
+      args:
+        git_sha: '${GIT_COMMIT}'
```

Comment:
tried to build this into the binary to no avail

Reply:
we still need this for docker builds since we don't have the full repo in the dockerfile